scale/storage/configuration/exceptions.py

```diff
@@ -0,0 +1,6 @@
+"""Defines exceptions that can occur when interacting with Workspace configuration"""
+
+
+class InvalidWorkspaceConfiguration(Exception):
```

Comment:
There was already an exception in /storage/brokers/exceptions for InvalidBrokerConfiguration. I think that InvalidBrokerConfiguration could be made a subclass of InvalidWorkspaceConfiguration.